### PR TITLE
Improvement translate into Russian

### PR DESCRIPTION
**Pull request by AndreyKozhev**
Improvement translate into Russian

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<resources xmlns:tools="http://schemas.android.com/tools">
+<resources tools:ignore="ExtraTranslation" xmlns:tools="http://schemas.android.com/tools">
```
Comment (itsaky) [inline app/src/main/res/values-ru/strings.xml:1]: Extra translations must be removed and not ignored. Please remove this ignore tag.

**Review by itsaky — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (AndreyKozhev) [reply app/src/main/res/values-ru/strings.xml]: Ready

**Review by itsaky — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -29,6 +29,7 @@
 - [ ] String Translator
 - [ ] Asset Studio (Drawable & Icon Maker)
 - [x] Git (*)
+- [ ] Plugins
```
Comment (itsaky) [inline README.md:32]: You're not supposed to add these yourself.

**Review by itsaky — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AndreyKozhev) [reply README.md]: OK

**Review by codejoker01 — APPROVED** [review]

**Review by itsaky — APPROVED** [review]

**Comment by itsaky** [comment]
Thank you for your contribution!

**Pull request merged**